### PR TITLE
Update cross-site service account documentation to OCP4

### LINKED ISSUE #1965: (Cross-site) Update service account documentation to OCP 4
Section https://infinispan.org/docs/infinispan-operator/main/operator.html#exchanging-sa-tokens_cross-site is outdated since it describes the process for OCP 3

In OCP 4, the procedure is different.

For non-expirable tokens:
https://docs.openshift.com/container-platform/4.14/nodes/pods/nodes-pods-secrets.html#nodes-pods-secrets-creating-sa_nodes-pods-secrets

For expiring tokens:
https://docs.openshift.com/container-platform/4.14/authentication/bound-service-account-tokens.html#bound-sa-tokens-configuring-externally_bound-service-account-tokens

Expiring tokens may require refreshing the secrets. Need to double-check that the operator can fetch the latest secret version/value.

### PR DESCRIPTION
**Pull request by pruivo**
Fixes #1965

### CODE REVIEW

File: documentation/asciidoc/topics/proc_exchanging_sa_tokens.adoc [hunk 1]
```diff
@@ -9,16 +9,42 @@ At **NYC** you add the service account token for **LON**.
...
+* Get the service account token.
++
+[source,options="nowrap",subs=attributes+]
 ----
+oc -n <namespace> get secrets ispn-xsite-sa-token -o jsonpath="{.data.token}" | base64 -d
+
+eyJhbGciOiJSUzI1NiIsImtpZCI6IiJ9
+----
+
+TIP: For bounded service account token, use the command `oc -n <namespace> create token <service account>`.
```
Comment (pruivo) [inline documentation/asciidoc/topics/proc_exchanging_sa_tokens.adoc:46]: would it make sense to create its own adoc file and sub-section for this type of token?

File: documentation/asciidoc/topics/proc_exchanging_sa_tokens.adoc [hunk 2]
```diff
@@ -6,35 +6,119 @@ After you create service account tokens on your {openshiftshort} clusters, you
...
+
+When a token expires, a new token needs to be created.
+Log in to the backup {openshiftshort} cluster and delete the secret `<token-secret>`.
+
+[source,bash,options="nowrap",subs=attributes+]
+----
+oc -n <namespace> delete secrets <token-secret>
+----
+
+Repeat this procedure to create a new token and generate a new `<token-secret>`.
```
Comment (ryanemerson) [inline documentation/asciidoc/topics/proc_exchanging_sa_tokens.adoc:119]: Is this sufficient, or do we also have to restart the Operator pods? I remember you mentioning this step in a meeting...

Comment (pruivo) [reply documentation/asciidoc/topics/proc_exchanging_sa_tokens.adoc:119]: The operator took about ~1 min to fetch the updated secret but we don't need to restart it :partying_face:

**Review by domiborges — APPROVED** [review]
I completed my review. Let's have two separate procedures.
Please let me know if it works out.

File: documentation/asciidoc/topics/proc_exchanging_bound_service_account_tokens.adoc [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+oc -n <namespace> create secret generic <token-secret> --from-literal=token=<token>
+----
++
+The `<token-secret>` is the name of the secret configured in the `Infinispan` CR.
+
+. Repeat the steps on other {openshiftshort} clusters.
+
+[discrete]
+== Deleting expired tokens
+When a token expires, delete the expired token, and then repeat the procedure to generate and exchange a new one.
```
Comment (pruivo) [inline documentation/asciidoc/topics/proc_exchanging_bound_service_account_tokens.adoc:50]:
```suggestion
When a token expires, delete the expired token secret, and then repeat the procedure to generate and exchange a new one.
```

File: documentation/asciidoc/topics/proc_exchanging_bound_service_account_tokens.adoc [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+[source,bash,options="nowrap",subs=attributes+]
+----
+oc -n <namespace> delete secrets <token-secret>
+----
+
+. Repeat the procedure to create a new token and generate a new `<token-secret>`.
+
+[role="_additional-resources"]
+.Additional resources
+* link:{link_os_sa_tokens}
```
Comment (pruivo) [inline documentation/asciidoc/topics/proc_exchanging_bound_service_account_tokens.adoc:65]: should link to bound service tokens. should I fix it? @domiborges

File: documentation/asciidoc/topics/proc_exchanging_service_account_tokens.adoc [hunk 5]
```diff
@@ -48,8 +50,12 @@ The command prints the token in the terminal.
...
 
 .Next steps
 
-* Repeat the preceding steps on your other {openshiftshort} clusters.
\ No newline at end of file
+* Repeat the preceding steps on your other {openshiftshort} clusters.
+
+[role="_additional-resources"]
+.Additional resources
+* link:{link_os_sa_bound_tokens}
```
Comment (pruivo) [inline documentation/asciidoc/topics/proc_exchanging_service_account_tokens.adoc:61]: same here

File: documentation/asciidoc/topics/proc_exchanging_bound_service_account_tokens.adoc (hunk 3 above)
Comment (domiborges) [reply documentation/asciidoc/topics/proc_exchanging_bound_service_account_tokens.adoc]: Updated

File: documentation/asciidoc/topics/proc_exchanging_bound_service_account_tokens.adoc (hunk 4 above)
Comment (domiborges) [reply documentation/asciidoc/topics/proc_exchanging_bound_service_account_tokens.adoc]: Correct. Now it's updated

**Comment by pruivo** [comment]
It looks good to me. Thanks @domiborges :+1: 
cc @ryanemerson

**Comment by ryanemerson** [comment]
Thanks @pruivo @domiborges

**Pull request merged**